examples/openfoam_supersonic_wedge.py

```diff
@@ -0,0 +1,249 @@
+"""
+Example which produces flow over a supersonic wedge
+
+>>> import os
+>>> case_dir = os.path.join(getfixture('tmpdir').strpath, 'cavity')
+>>> main(case_dir)
+>>> os.path.isdir(os.path.join(case_dir, '10'))
+True
+
+"""
+
+import os
+from cusfsim.case import (
+    Case, Dimension, FileName, FileClass
+)
+
+def main(case_dir='wedge'):
+    #Try to create new case directory
+    case = create_new_case(case_dir)
+    # Add the information needed by blockMesh.
+    write_initial_control_dict(case)
+    write_block_mesh_dict(case)
+    #we generate the mes1h
+    case.run_tool('blockMesh')
+    #we prepare the thermophysical and turbulence properties
+    write_thermophysical_properties(case)
+    write_turbulence_properties(case)
+    #we write fvScheme and fvSolution
+    write_fv_schemes(case)
+    write_fv_solution(case)
+    write_initial_conditions(case)
+    case.run_tool('rhoCentralFoam')
+
+def create_new_case(case_dir):
```

Comment:
I wonder if this functionality should be added to cusfsim.Case proper. It's consistent with other APIs which make it hard to accidentally overwrite things. Opinions?